devops/poseidon-deployer/test/poseidon.test.ts

```diff
@@ -0,0 +1,77 @@
+import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { ethers } from "hardhat";
+import hre from "hardhat";
+import {writeFileSync, unlinkSync} from "fs";
+
+import { verifyPoseidonContracts } from "../tasks/utils";
+
+const { buildPoseidonOpt } = require('circomlibjs');
+
+
+const deployPoseidons = async (muticall: boolean, factoryAddress?: string, salt?: string) => {
+  let poseidonAddressBook;
+  switch (muticall) {
+    case true:
+      poseidonAddressBook = await hre.run("deploy-poseidon", {multicall: true, factoryAddress: factoryAddress, salt: salt});
+      break;
+    default:
+      poseidonAddressBook = await hre.run("deploy-poseidon");
+  }
+  return poseidonAddressBook;
+}
+
+const deployPoseidonFactory = async (poseidonAddressBook: any) => {
+  const poseidonAddressBookFilePath = './PoseidonAddressBook.test.toml';
+  // `contents` follows TOML syntax
+  const contents = 
+      `[poseidonAddresses]
+      T2 = "${poseidonAddressBook.get("T2")}"
+      T3 = "${poseidonAddressBook.get("T3")}"
+      T4 = "${poseidonAddressBook.get("T4")}"
+      T5 = "${poseidonAddressBook.get("T5")}"
+      T6 = "${poseidonAddressBook.get("T6")}"
+      T7 = "${poseidonAddressBook.get("T7")}"`
+      
+      writeFileSync(poseidonAddressBookFilePath, contents);
```

Comment:
Run linter

Reply:
Had run prettier and ESLint in 3f753a1792931a5910c3614db2426bef2bb134c1